packages/remix-server-runtime/data.ts

```diff
@@ -34,7 +34,10 @@ export async function callRouteAction({
   isRemixRouterRequest?: boolean;
 }) {
   if (!action) {
-    let response = new Response(null, { status: 405 });
+    let response = new Response(null, {
+      status: 405,
+      statusText: "Method Not Allowed",
```

Comment:
Add `statusText` to this Remix-generated Response to align with 405's from elsewhere (including the router)